app/controllers/books_controller.rb

```diff
@@ -4,13 +4,11 @@ class BooksController < ApplicationController
   before_action :set_book, only: %i[show edit update destroy]
 
   # GET /books
-  # GET /books.json
   def index
-    @books = Book.all
+    @books = Book.order('created_at DESC').page(params[:page]).per(10)
```

Comment:
`order` は文字列でSQLを指定するよりも、シンボルを指定するやり方をおすすめします。

文字列の場合はDBにSQLをそのまま渡してしまいますが、シンボルであればRailsがSQLを適切に組み立ててくれます。

https://api.rubyonrails.org/v7.0/classes/ActiveRecord/QueryMethods.html#method-i-order

挙動の違いなどは、こちらのリファレンスを参照してみてください。

```suggestion
    @books = Book.order(created_at: :desc).page(params[:page]).per(10)
```